dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,3 +24,29 @@ properties:
     type: phandle-array
     required: false
     description: RESETn pin
+  sfdp-basic-8:
```

Comment:
I believe these should move to `jedec,spi-nor-common.yaml` where `has-be32` used to be, since they are generic and independent of what driver is used to access the device (SPI or QSPI).

Reply:
I have put the bindings in `jedec,spi-nor.yaml` because I can not see a way to implement the features for `nrf_qspi_nor.c`. The erase function is part of the nrf hal (I guess) and doesn't take an erase cmd as argument. I can not find any declaration for `nrfx_qspi_erase` in the zephyr tree.

Reply:
You don't need to implement using those properties in this PR.  (`nrfx_qspi_erase` is in the Nordic HAL, and it figures out what it needs to do.  The peripheral might even issue multiple commands, I don't remember.)

My point is that these are properties of the device, not the interface, so they're common to all interfaces.  There are properties currently in `nordic,qspi-nor` that are described in other SFDP words; in the future it would be good to move them to that interface for consistency, and putting them there now saves moving them later.

Reply:
Thanks. Done